### PR TITLE
Fix PID and filter slider update after importing invalid configuration

### PR DESCRIPTION
**Pull request by haslinghuis**
After merging https://github.com/betaflight/betaflight-configurator/pull/2676 we have a problem which nobody noticed.

Problem: Change a slider and leave tab. Come back to configurator. Now the slider is disabled.
Cause: On loading tab TuningSliders are not initialized yet, but some jquery handlers were using the TuningSliders multiplier values already.

Fixed handling of importing invalid configurator changes. (Sliders are left on but values are changes manually)

Depends on:
https://github.com/betaflight/betaflight/pull/11104 1st commit
https://github.com/betaflight/betaflight/pull/11114 2nd commit

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
This fixes the problems we currently have with changing the PID slider mode to ON/OFF, and when moving any slider multiplier.

However I found that if I go to CLI and turn a filter slider OFF, on returning to Configurator I find it back 'ON' again, and it gets saved to 'OFF'.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
@ctzsnooze Fixed. Removed double initialization.

File: src/js/tabs/pid_tuning.js
Comment (limonspb) [inline src/js/tabs/pid_tuning.js:957]: Hi Mark.
This code calculates gyro lowpass2 cutoff based on the slider value.
However, this code is inside 4.3 section. And for 4.3 the Configurator should not be calculating cutoffs like that. Instead, it should be taking the values from the firmware.

The current implementation here leads to the following misconception:
In 4.3 the Configurator should not be in charge of calculating values by itself.
The firmware must tell to the configurator the values.

Also it creates the following bug. Steps:
1. start from defaults - sliders ON
2. CLI: set gyro_lpf2_static_hz = 666
3. Open UI - the value is shown as 500 instead of 666 because it has been calculated in the configurator.

I believe in this PR i see other places where Configurator calculates the filter values based on sliders.

**Review by limonspb — CHANGES_REQUESTED** [review]

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [reply src/js/tabs/pid_tuning.js]: @limon this code deals with enabling slider switches making sure they align with the chosen slider value.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
After a LOT of work, this now is tested and works well.  Much gratitude to @haslinghuis and @limonspb.

This fixes issues with the Refresh button.  It now reliably restores slider position and mode settings, and PID values.

In case of conflict between existing/saved PID numbers, and values from slider settings, the PID numbers will not be erased unless the user saves with the slider active.  

Conflicts of that kind are uncommon, but can arise from pasted CLI snippets, OSD/LUA changes, or in-flight adjustments.  https://github.com/betaflight/betaflight/pull/11111 should solve the OSD problem, but pasted snippets remain a concern.

We strongly recommend using sliders to tune in 4.3.  

Before adjusting PIDs using direct numeric entry methods, it's important to first disable the relevant sliders.

**Review by ctzsnooze — DISMISSED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by Asizon — CHANGES_REQUESTED** [review]
@haslinghuis I can see for old versions like 4.2 that changing derivative or feedforward manualy doesnt disable sliders. It works for P,I and Dmax.

**Pull request closed without merging**
